src/group_theory/sylow.lean

```diff
@@ -214,7 +214,7 @@ have hm' : p ∣ card (quotient (subgroup.comap ((normalizer H).subtype : normal
   nat.dvd_of_mod_eq_zero
     (by rwa [nat.mod_eq_zero_of_dvd (dvd_mul_left _ _), eq_comm] at hm),
 let ⟨x, hx⟩ := @exists_prime_order_of_dvd_card _ (quotient_group.group _) _ _ hp hm' in
-have hxcard : ∀ {f : fintype (gpowers x)}, card (gpowers x) = p,
+have hxcard : ∀ {f : fintype (subgroup.gpowers x)}, card (subgroup.gpowers x) = p,
```

Comment:
Is there a reason you added this change here? It still works for me if I remove the `subgroup.` part of the name.
```suggestion
have hxcard : ∀ {f : fintype (gpowers x)}, card (gpowers x) = p,
```

Reply:
I got ambiguous name overloads without it.

Reply:
See https://github.com/leanprover-community/mathlib/runs/1009758832#step:7:10

Reply:
Let's keep the disambiguation then. Weird that it works for me though.